usr/share/lib/img_proof/tests/SLES/Azure/test_sles_azure_csp_cli.py

```diff
@@ -1,26 +1,20 @@
 import pytest
-import re
 
-def test_sles_azure_csp_cli(host, get_release_value, is_sle_micro):
-    version = get_release_value('VERSION')
-    assert version
+
+def test_sles_azure_csp_cli(host, is_sle_micro, get_version):
+    version = get_version()
 
     if is_sle_micro():
         pytest.skip('Micro has product version instead of SLE version.')
 
-    match = re.match(r'^(\d+)(?:[.-](?:SP)?(\d+))?$', version)
-    assert match, f"Unexpected version format: {version}"
-
-    major = match.group(1)
-    patchlevel = match.group(2)
-
-    if int(major) < 16:
+    if version < 16.0:
```

Comment:
But now we skip the test for anything SLE 15 and SLE 12 and those have, are supposed to have the cli tools in the image. the cli tools.

Reply:
Confirm test for 15 and 12. Will include a change in the follow on PR.